packages/sqs/test/consumers/SqsPermissionConsumer.spec.ts

```diff
@@ -596,6 +597,66 @@ describe('SqsPermissionConsumer', () => {
     })
   })
 
+  describe('multiple consumers', () => {
+    let diContainer: AwilixContainer<Dependencies>
+    let sqsClient: SQSClient
+
```

Comment:
🟢 I would suggest adding a simple test to `AbstractSnsSqsConsumer` just to make sure future changes are not breaking multiple consumer support

Reply:
Similar test added for `AbstractSnsSqsConsumer`